src/renderer/components/WatchVideoDescription/WatchVideoDescription.vue

```diff
@@ -1,19 +1,49 @@
 <template>
   <FtCard
     v-if="shownDescription.length > 0"
-    class="videoDescription"
+    :class="{ videoDescription: true, short: !showFullDescription }"
   >
     <FtTimestampCatcher
+      ref="descriptionContainer"
       class="description"
       :input-html="shownDescription"
       @timestamp-event="onTimestamp"
     />
+    <template v-if="showControls">
+      <span
+        v-if="showFullDescription"
+        class="descriptionStatus"
+        tabindex="0"
+        @click="collapseDescription"
+        @keydown.space.prevent="collapseDescription"
+        @keydown.enter.prevent="collapseDescription"
+      >
+        {{ $t("Description.Collapse Description") }}
+      </span>
+      <span
+        v-else
+        class="descriptionStatus"
+        tabindex="0"
+        @keydown.space.prevent="expandDescription"
+        @keydown.enter.prevent="expandDescription"
+      >
+        {{ $t("Description.Expand Description") }}
+      </span>
```

Comment:
This is working great through tab navigation! Last comment; could you add role="button" to these two?

Reply:
My bad! I accidentally removed those, just pushed that now.